rebar.config

```diff
@@ -461,8 +461,25 @@
 			%% Your mining address will be initialized with this amount of AR when you
 			%% launch your localnet.
 			%% {d, 'LOCALNET_BALANCE', 1_000_000_000_000}
+			export_all,
+			no_inline
+		]},
+		{overrides, [
+				{override, arweave, [
+						{deps, [
+								{recon, {git, "https://github.com/ferd/recon.git", {tag, "2.5.6"}}}
```

Comment:
Recon can be used to trace the function calls. The only problem is that it only can trace exported functions. So in case of localnet we're dropping a few optimisations to obtain some transparency for tracing.
See the [recon_trace](https://ferd.github.io/recon/recon_trace.html) documentation.
Actually, recon is safe to use in production, nut I didn't want to introduce any unnecessary dependencies iwhtout discussion.